app/assets/stylesheets/user-profile-header.scss

```
@@ -282,6 +282,10 @@
         display: inline-block;
         font-size: 0.85em;
         margin-bottom: 4px;
+
+        a {
+          color: var(--button-primary-bg);
```

I believe this color is actually branded... Meaning it always inherits colors from user's accent color that they specified in settings.

If we're comfortable with overwriting that, let's use other - more appropriate - variable and add hover state as well:

```suggestion
          color: var(--link-brand-color);
          
          &:hover {
              color: var(--link-brand-color-hover);
          }
```

At the same time we should probably get rid of this guy: https://github.com/thepracticaldev/dev.to/blob/a2b7415836d7a29801e777ba8337b09df1314019/app/views/articles/_user_metadata.html.erb#L1-L5

---

ooh! that's why I didn't have the same initial color as this user https://github.com/thepracticaldev/dev.to/issues/4879#issuecomment-604208514

---

We should probably park this PR until we know what everyone wants to do about that color branding. Thanks a lot @ludwiczakpawel!

---

@rhymes If this is still parked, can you maybe add [WIP] to the title? I just got a pull reminder for this on Slack.